src/profile-logic/data-structures.js

```diff
@@ -336,6 +336,7 @@ export function getDefaultCategories(): CategoryList {
     { name: 'Network', color: 'lightblue', subcategories: ['Other'] },
     { name: 'Graphics', color: 'green', subcategories: ['Other'] },
     { name: 'DOM', color: 'blue', subcategories: ['Other'] },
+    { name: 'Native', color: 'blue', subcategories: ['Other'] },
```

Comment:
Instead of changing the default data structure, I'd suggest to just change it in src/profile-logic/import/chrome.js. That would be right after `const profile = getEmptyProfile();` in `processTracingEvents`. We could add only the categories that are useful for this importer instead of having them all. Something such as:
```js
profile.meta.categories = [
    { name: 'Other', color: 'grey', subcategories: ['Other'] },
    { name: 'Idle', color: 'transparent', subcategories: ['Other'] },
    { name: 'JavaScript', color: 'yellow', subcategories: ['Other'] },
    { name: 'GC / CC', color: 'orange', subcategories: ['Other'] },
    { name: 'Native', color: 'blue', subcategories: ['Other'] },
];
```

With this change I believe the tests won't need to be changed besides `profile-conversion.test.js.snap`.

Reply:
Thanks for the suggestion! Lgtm 8c8288b5e258d4a8c85d3e45ba05b79aa9770b1f, except that chrome.js requires that the Graphics category be present for some reason, and yes it's definitely better if color palettes for different purposes are not mixed up together.

Reply:
ah indeed, the Graphics category is used for the screenshot markers. We don't do much with that information yet (besides sorting in the marker chart) but in the future this may change the marker's color in some situation.